lib/hybrid-sdk/client/dispatcher/client/api.ts

```diff
@@ -63,7 +65,7 @@ export class HttpDispatcherServiceClient implements DispatcherServiceClient {
       return serverId;
     } catch (err) {
       logger.trace({ err }, 'Error getting connection allocation.');
-      throw new Error('Error getting connection allocation.');
+      throw new Error(`Error getting connection allocation.${err}`);
```

Comment:
nitpick, whitespace between `allocation.` and `${err}`?

Reply:
done